articles/imagesharp.web/processingcommands.md

```diff
@@ -80,3 +90,11 @@ Allows the changing of the background color of transparent images.
 {PATH_TO_YOUR_IMAGE}?bgcolor=128,64,32
 {PATH_TO_YOUR_IMAGE}?bgcolor=128,64,32,16
 ```
+
+#### Auto Orient
+
+Allows the normalization of image orientation based on the source EXIF metadata.
+
+```
+{PATH_TO_YOUR_IMAGE}?autoorient=true
```

Comment:
Should this be `orient` instead?

Reply:
No. `orient` is a command tied to `Resize` (defaulting to `true`) that will swap width/height commands for EXIF rotated images. If you pass `false` to `Resize` you can disable the functionality.